app/views/manuals/show.html.erb

```diff
@@ -99,8 +99,11 @@
   </div>
 
   <div class="govuk-grid-column-one-third">
-    <% if allow_publish?(manual, slug_unique) || !manual.has_ever_been_published? %>
+    <% if allow_publish?(manual, slug_unique) || !manual.has_ever_been_published? || publish_text(manual, slug_unique) != "" %>
```

Comment:
A minor gripe, but feels like this conditional is getting a bit lengthy, especially as we then check most of the conditions again inside it. I wonder if there's a way to model the conditions a little more effectively? We have [a component](https://github.com/alphagov/whitehall/blob/main/app/components/admin/editions/show/sidebar_actions_component.rb) for this in Whitehall, I know Manuals doesn't use components but maybe building the markup in the helper rather than the template would feel cleaner?